src/index.spec.js

```diff
@@ -1,27 +1,222 @@
+import mongodb from 'mongodb';
```

Comment:
Do you have an idea about how to write the two tests without having to duplicate all tests functions ?